cmake/compiler_config.cmake

```diff
@@ -99,6 +99,12 @@ function(mayaUsd_compile_config TARGET)
             PRIVATE
                 ${GNU_CLANG_FLAGS}
         )
+        if(IS_LINUX)
+            target_compile_definitions(${TARGET}
+                PRIVATE
+                    _GLIBCXX_USE_CXX11_ABI=0 # USD is built with old ABI
```

Comment:
Is it true for every USD artifact or there can be USD artifacts (think of a custom build at a studio) that build with a new ABI?

Reply:
Maya is using old ABI and has the same define defined in its CMake files.